app/api/organizations/route.ts

```diff
@@ -2,6 +2,33 @@ import { NextRequest, NextResponse } from 'next/server'
 import OrganizationController from '@/server/controller/organization'
 import { AddOrUpdateFile } from '@/server/utils/github'
 import Session from '@/utils/session'
+import EventController from '@/server/controller/event'
+import { IEvent } from '@/server/model/event'
+import { IOrganization } from '@/server/model/organization'
+import { generateId } from '@/server/utils'
+
+/*
+ * The events of the organization will also be deleted
+ */
+export async function DELETE(req: NextRequest): Promise<NextResponse> {
```

Comment:
Need to check Session

Reply:
The same is probably true for any method with POST, PUT or DELETE